lighty-modules/lighty-gnmi/lighty-gnmi-commons/src/main/java/io/lighty/modules/gnmi/commons/util/DataConverter.java

```diff
@@ -232,16 +233,26 @@ public static Optional<Module> findModuleByQName(@NonNull final QName element,
     }
 
     public static Optional<DataSchemaNode> findAugmentationDataNode(@NonNull final String element,
-                                                                    @NonNull final EffectiveModelContext context) {
+            @NonNull final EffectiveModelContext context, @NonNull final YangInstanceIdentifier parentYIID) {
```

Comment:
Your code will not work because you are not using provided parameter `String element` which is local-name for searched element. In current stage, we only know parent YIID and local-name for search node. If searched node is augmented from different yang model than parent YIID we also don't know namespace and revision.
This part of code will find augmentation with same YIID as parent YIID. If this augmentation exist, we will look for child with same local-name as provided element, Child with same local-name is DataSchemaNode which we are looking for.